torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1748,6 +1749,31 @@ RegisterOperators reg2({
     DEFINE_INT_OP(aten::__or__, a | b),
     DEFINE_INT_OP(aten::__xor__, a ^ b),
 
+    Operator(
+        "prim::abs(int x) -> int",
+        [](Stack& stack) {
+          int64_t x;
+          pop(stack, x);
+          push(stack, static_cast<int64_t>(std::abs(x)));
```

Comment:
Sorry, I didn't realize std::abs wasn't defined for ints as well. Could you put it back to what you had previously?

Reply:
It's defined for ints as well, the cast was just plain unnecessary.